app/javascript/article-form/articleForm.jsx

```diff
@@ -16,6 +16,7 @@ import ImageManagement from './elements/imageManagement';
 import MoreConfig from './elements/moreConfig';
 import OrgSettings from './elements/orgSettings';
 import Errors from './elements/errors';
+import HotkeyLinstener from './elements/hotkeyLinstener';
```

Comment:
I think there's a tytypo in the name `HotkeyLinstener`, it should probably be `HotKeyListener` but I'd call it something like `keyboardShortcutsHandler`, it's easier to find a few weeks down the road :D

This could be shared for comment editor as well, nice.

Reply:
Yes, It's a typo. 
`KeyboardShortcutsHandler` sounds better for me.

Reply:
I changed it to `KeyboardShortcutsHandler` ;)

should I implement hotkey for comment editor as well for this pull request ?

Reply:
@Elecweb no, I think it belongs in a separate PR, maybe after opening an issue about it :)

Reply:
I wonder if you can add a GIF demo video, I use [Licecap](https://www.cockos.com/licecap/) on macOS for example

Reply:
@rhymes sure ;)